usefulUtilities/voteApp/appResources/voteApp.js

```diff
@@ -0,0 +1,1022 @@
+"use strict";
```

Comment:
[coding standards](https://highfidelity.atlassian.net/wiki/spaces/CPT/pages/442073291/Coding+Standards): throughout the file, i'd like to see comments above each function describing what it does.